SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -219,6 +219,18 @@ void USpatialGameInstance::HandleOnConnected()
 	WorkerConnection->OnEnqueueMessage.AddUObject(SpatialLatencyTracer, &USpatialLatencyTracer::OnEnqueueMessage);
 	WorkerConnection->OnDequeueMessage.AddUObject(SpatialLatencyTracer, &USpatialLatencyTracer::OnDequeueMessage);
 #endif
+	// Cleanup any actors which were created during level load.
+	UWorld* World = GetWorld();
+	check(World != nullptr);
+	for (ULevel* Level : Levels)
```

Comment:
Perhaps the `Levels` variable could be slightly more expressive. eg. `CachedLevelsForNetworkIntialize` ?

Reply:
Good idea.